README.md

```diff
@@ -2,74 +2,17 @@
 
 # Project Name
 
-> One paragraph statement about the project.
+> Hello-Microverse!
 
-![screenshot](./app_screenshot.png)
-
-Additional description about the project and its features.
+Set up a "Hello Microverse" project
```

Comment:
Kindly work on these changes:
- Your readme file is missing general information about the creator, descriptive information about on how to download and install your project locally, once installed how to run it, which requirements are needed before to use it probably in this case isn't needed to have information about testing and deployment.
- You can use [this template.](https://github.com/microverseinc/readme-template)

Reply:
Hi Carlos, thank you for your review; I created a short version of the README because this was only a Hello Microverse! Project. And since the last reviewer did not mention the README in their review, I assumed that it was okay.